Added change-log for new GA features (#4314)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/cumulocity-2025.373.0-dashboard-advanced-tab-ga.md

---
date:

> title: Advanced dashboard moved to General Availability

Reviewer: ```suggestion
title: Advanced dashboard feature moved to General Availability
```

> title: Dashboard advanced tab moved to General Availability

Reviewer: ```suggestion
title: Advanced dashboard moved to General Availability
```

> title: Advanced dashboard feature moved to General Availability

Reviewer 2: ```suggestion
title: Dashboard Import/Export feature moved to General Availability
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-65134
version: 2025.373.0
---

> The Dashboard advanced tab, which was previously available as Public Preview, has now been promoted to General Availability (GA). This feature enables users to easily import and export dashboards, facilitating dashboard sharing and backup across different tenants and environments. With GA status, this functionality is now fully production-ready and supported, allowing users to confidently leverage dashboard import and export capabilities in production environments with full stability guarantees.

Reviewer: ```suggestion
The advanced dashboard feature, which was previously available as Public Preview, has now been promoted to General Availability (GA). This feature enables users to easily import and export dashboards, facilitating dashboard sharing and backup across different tenants and environments. With GA status, this functionality is now fully production-ready and supported, allowing users to confidently leverage dashboard import and export capabilities in production environments with full stability guarantees.
```

> The advanced dashboard feature, which was previously available as Public Preview, has now been promoted to General Availability (GA). This feature enables users to easily import and export dashboards, facilitating dashboard sharing and backup across different tenants and environments. With GA status, this functionality is now fully production-ready and supported, allowing users to confidently leverage dashboard import and export capabilities in production environments with full stability guarantees.

Reviewer: ```suggestion
The advanced dashboard feature, which was previously available as Public Preview (see [announcement](/change-logs/#ui-c8y-1022-0-0-dashboard-cross-application-copy-paste), has now been promoted to General Availability (GA). This feature enables users to easily import and export dashboards, facilitating dashboard sharing and backup across different tenants and environments. With GA status, this functionality is now fully production-ready and supported, allowing users to confidently leverage dashboard import and export capabilities in production environments with full stability guarantees.
```

> The advanced dashboard feature, which was previously available as Public Preview (see [announcement](/change-logs/#ui-c8y-1022-0-0-dashboard-cross-application-copy-paste), has now been promoted to General Availability (GA). This feature enables users to easily import and export dashboards, facilitating dashboard sharing and backup across different tenants and environments. With GA status, this functionality is now fully production-ready and supported, allowing users to confidently leverage dashboard import and export capabilities in production environments with full stability guarantees.

Reviewer 2: ```suggestion
The dashboard import/export feature, previously available as Public Preview (see [announcement](/change-logs/#ui-c8y-1022-0-0-dashboard-cross-application-copy-paste)), is now promoted to General Availability (GA).

Dashboards can be exported as JSON files including all metadata such as widget configurations and settings, and imported across tenants and environments.

When importing dashboards between different contexts (e.g., devices to assets or across tenants), a configuration review is prompted to adjust settings and ensure compatibility.
```
